docs/index.rst

```diff
@@ -739,6 +739,20 @@ its renderers are added to the config and can be used.::
             # templates/home.pt starts with the standard <html> tag for HTML5
             self.assertTrue('<html' in response.body)
 
+Precompiling
+------------
+
+When first rendered, a Chameleon template is "compiled" to Python and
+then the resulting code is executed. The compiled Python code may then
+be stored to disk in a cache directory set in the `CHAMELEON_CACHE`
```

Comment:
reStructuredText will render this in italics, whereas markdown renders it as inline code. I think you want this rendered as inline code. To do so, use double-ticks:

```rst
``CHAMELEON_CACHE``
```